inference_perf/loadgen/load_timer.py

```diff
@@ -89,3 +90,22 @@ def start_timer(self, initial: Optional[float] = None) -> Generator[float, None,
             for _ in range(req_count):
                 next_time = next(time_generator)
                 yield next_time
+
+class TraceReplayLoadTimer(LoadTimer):
+    def __init__(self, trace_reader: TraceStreamReader, trace_file: Path, has_header: bool = False) -> None:
```

Comment:
how is has_header intended to be processed / identified? Is this something that load timer / trace_reader can handle automatically?